### PR TITLE
Feature/header component

### PR DESCRIPTION
**Pull request by apete12**
What I did: Add header component with styling. 

Testing Notes/Extras: 

Did this break anything?
- [X]  No
- [ ]  Yes

Type of change
- [ ]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Added testing for: 
- [X]  Skip all the other stuff and briefly explain the fix: header component with styling. 

Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [X]  The code runs locally
- [X]  The code is DRY. If it's not, I tried my best
- [X]  I reviewed my code before pushing

What's next?
update CSS with new color palette.

### CODE REVIEW

**Review by aliceabarca — COMMENTED** [review]
Awesome job with the header!

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
-import logo from './logo.svg';
 import './App.css';
-import Form from './components/Forms/Form';
```
Comment (paulina-isabel) [inline src/App.js:3]: Is it ok if we change the "Components" file name to "components" with all lowercase? I think convention is to only capitalize the first letter of an actual component. Hope that makes sense.

File: src/components/Header/Header.js
Comment (paulina-isabel) [inline src/components/Header/Header.js:1]: Love the emoji use here!

**Review by paulina-isabel — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (apete12) [reply src/App.js:3]: Yes! Just made that change, thanks for catching that!

File: src/index.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Abel&family=Abril+Fatface&family=Aleo:wght@200&family=Antic+Didone&family=Bebas+Neue&family=Cormorant+Garamond:wght@300&family=Oswald:wght@500&family=Raleway:wght@100;400;600&display=swap');
```
Comment (judy0ye) [inline src/index.css:1]: the font looks great Alex!

**Pull request merged**